### PR TITLE
add display_source field to news-flash output

### PR DESCRIPTION
**Pull request by advak**
Signed-off-by: Adva Klinger <advaklinger@gmail.com>

### CODE REVIEW

File: anyway/flask_app.py [hunk 1]
```diff
@@ -381,6 +381,10 @@ def yishuv_symbol_to_name():
     return Response(json.dumps(entries, default=str), mimetype="application/json")
 
 
+source_mapping = {"walla": "וואלה",
```
Comment (atalyaalon) [inline anyway/flask_app.py:384]: Perhaps adding this to constants.py file?

**Comment by atalyaalon** [comment]
@advak great work, added a small comment :)

File: anyway/flask_app.py (hunk 1 above)
Comment (advak) [reply anyway/flask_app.py]: @atalyaalon, thanks for your comment. Totally agree. Fixed and pushed to the PR

**Comment by atalyaalon** [comment]
> The PR is good and straight forward, and could be merged as is 😃 .
> I got a few suggestion that might be worth considering:
> 
> 1. `news_flash` Is already a huge function. I think that any functionality added to it should be in auxiliary function, Once we will have such function it will be a lot easier to test the added functionality.
> 2. I think you should handle `KeyError` when going through the mapping.

Great comments, @advak what do you think?

**Comment by advak** [comment]
@idosavion, thanks for your comments. I agree that the more functionality we have in news-falsh it will be better to divide it to some auxiliary functions. Regarding the KeyError, do you mean to add try, except logic when getting the source mapping? I can implement it in this commit

**Comment by atalyaalon** [comment]
@advak note there is a conflict in anyway/constants.py

**Comment by advak** [comment]
@atalyaalon , it's fixed

**Pull request merged**